src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs

```diff
@@ -21,24 +21,25 @@ public class PackageManagerSearchElementViewModel : BrowserItemViewModel, IEquat
         public ICommand VisitRepositoryCommand { get; set; }
         public ICommand DownloadLatestToCustomPathCommand { get; set; }
 
-        /// <summary>
-        /// A function used to determine whether the user has installed this package already.
-        /// Matching occurs via package name.
-        /// </summary>
-        private Func<string,bool> isPackageInstalled;
-
         public new PackageManagerSearchElement Model { get; internal set; }
 
-
-        public PackageManagerSearchElementViewModel(PackageManagerSearchElement element, bool canLogin) : base(element)
+        /// <summary>
+        /// Alternative constructor which takes a Function object in order to
+        /// assist communication between the PackageManagerSearchViewModel
+        /// and the PackageManagerSearchElementViewModel.
+        /// </summary>
+        /// <param name="element">A PackageManagerSearchElement</param>
+        /// <param name="canLogin">A Boolean used for access control to certain internal packages.</param>
+        /// <param name="canInstall">Whether a package can be installed.</param>
+        internal PackageManagerSearchElementViewModel(PackageManagerSearchElement element, bool canLogin, bool canInstall) : base(element)
```

Comment:
Made this internal. I hope we can do that in between releases...

Reply:
@mjkkirschner ?

Reply:
hmm I think constructors are usually public, any reason to force `internal`?

Reply:
@pinzart90 - no sorry, if this was actually part of a public class, and it was public, and released, then making it internal is like removing the method.

Reply:
I guess I'm confused by what you mean by "between releases"

Reply:
@QilongTang correction
The public CanInstall property has an internal setter.
Do you guys still think I should make the Constructor public ?

Reply:
I made it public